.pre-commit-config.yaml

```diff
@@ -39,7 +39,7 @@ repos:
     hooks:
       - id: generate-manifests
         name: Generate Manifests
-        entry: make generate manifests
+        entry: make generate manifests build-installer
```

Comment:
For troubleshooting purposes, I believe it would be clearer to have separate hooks for each case. This way, it’s easier to identify which make command failed.